source/layouts/layout.haml

```diff
@@ -46,24 +46,26 @@
         = link_to 'upgrade your browser', 'https://browsehappy.com/'
         to improve your experience and security.
 
-    .sl-c-alert.sl-c-alert--info
-      .sl-l-container
-        %p
-          %strong Sass just launched a brand new module system.
-          = succeed '!' do
-            = link_to '/blog/posts/7858341-the-module-system-is-launched' do
-              Learn all about it on the Sass blog
-
-    .sl-c-pop-stripe
-
+    %a.skip-link(href='#main-content')
+      Skip to content
+        
     %header.sl-r-banner(itemtype='http://schema.org/WPHeader' itemscope='itemscope' role='banner')
+      .sl-c-alert.sl-c-alert--info
+        .sl-l-container
+          %p
+            %strong Sass just launched a brand new module system.
+            = succeed '!' do
+              = link_to '/blog/posts/7858341-the-module-system-is-launched' do
+                Learn all about the new on the Sass blog
```

Comment:
This should remain as "Learn all about it on the Sass blog".

Reply:
I can revert it, but to provide context: I changed it so that if someone gathers all links all links from the page (which is a common thing that screen reader users do), it's not clear what they're learning.

Let me know how you'd like me to proceed.

Reply:
I see. In that case, "Learn all about the module system on the Sass blog" would be more accurate.

Reply:
Excellent. Will update with next PR. 👍🏽